src/inputs/Button/button.stories.tsx

```diff
@@ -176,9 +174,7 @@ export const withCustomComponent = (): React.ReactElement => {
       color="secondary"
       component={Link}
```

Comment:
There is an issue with component after the refactor. Is not recognized as a valid property

Reply:
@dasanra good catch! it should be fixed now!